utils/general.py

```diff
@@ -53,6 +54,25 @@ def download_file_blob_to_dir(directory, file_blob):
     os.remove(file_destination)
 
 
+def move_downloaded_data_to_single_directory(directories):
+
+    for directory in directories:
+        if not os.path.isdir(directory + "train"):
+            os.makedirs(directory + "train")
+        if not os.path.isdir(directory + "val"):
+            os.makedirs(directory + "val")
+
+        folders = os.listdir(directory)
+        for folder in folders:
+            if folder != "train" and folder != "val":
```

Comment:
Checks like this can be written as 
```python
if folder not in ["train", "val"]:
    ...
```
in python.

Reply:
Fixed c24645245cae0f091301b47eb829857fa13fca68